src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/TransformationToolbar.java

```diff
@@ -31,89 +31,52 @@
-    try {
-      WaitHelper.waitForPageToLoad();
-      WaitHelper.waitForElementToBeDisplayed(Helper.locateElementByTestId("transformations-toolbar-container"));
-      Assert.assertTrue(Helper.isElementExists
-      (Helper.getCssSelectorByDataTestId("transformations-toolbar-container")));
-    } catch (Exception e) {
-      System.err.println("error:" + e);
-    }
+    WaitHelper.waitForPageToLoad();
+    WaitHelper.waitForElementToBeDisplayed(Helper.locateElementByTestId("transformations-toolbar-container"));
+    Assert.assertTrue(Helper.isElementExists(Helper.getCssSelectorByDataTestId("transformations-toolbar-container")));
   }
   @Then("Verify if all icons are displayed on Toolbar with \\\"(.*)\\\"")
   public void verifyAllTheIconsAreDisplayedOnToolbar(String testId) {
-    try {
-      WaitHelper.waitForPageToLoad();
-      WaitHelper.waitForElementToBeDisplayed(Helper.locateElementByTestId("toolbar-icon-"
-              + testId));
-      Assert.assertTrue(Helper.isElementExists(Helper.getCssSelectorByDataTestId("toolbar-icon-"
-                    + testId)));
-    } catch (Exception e) {
-      System.err.println("error:" + e);
-    }
+    WaitHelper.waitForPageToLoad();
+    WaitHelper.waitForElementToBeDisplayed(Helper.locateElementByTestId("toolbar-icon-"
+        + testId));
+    Assert.assertTrue(Helper.isElementExists(Helper.getCssSelectorByDataTestId("toolbar-icon-"
+        + testId)));
   }
   @Then("Click on the function names toggle with testId as \\\"(.*)\\\" and \\\"(.*)\\\"")
   public void clickOnTheSliderButton(String testId, String iconLabelName) {
-    try {
-      Helper.waitSeconds(5);
-      WaitHelper.waitForElementToBeEnabled
-              (Helper.locateElementByTestId("transformations-toolbar-icons-function-name-toggler"));
-      WebElement toggle = Helper.locateElementByTestId("transformations-toolbar-icons-function-name-toggler");
-      WaitHelper.waitForElementToBeDisplayed(toggle);
-      WebElement ele = Helper.locateElementByXPath
-        ("//input[@data-testid='transformations-toolbar-icons-function-name-toggler']");
-      Actions action = new Actions(SeleniumDriver.getDriver());
-      WaitHelper.waitForPageToLoad();
-      action.moveToElement(ele).perform();
-      ele.click();
-      WaitHelper.waitForPageToLoad();
-      WebElement icon = Helper.locateElementByTestId("toolbar-icon-title-" + testId);
-      String text = icon.getText();
-      String actual = iconLabelName;
-      Assert.assertEquals(text, actual);
-    } catch (Exception e) {
-      System.err.println("error: " + e);
-    }
+    Helper.waitSeconds(5);
+    WaitHelper.waitForElementToBeEnabled
+    (Helper.locateElementByTestId("transformations-toolbar-icons-function-name-toggler"));
+    WebElement toggle = Helper.locateElementByTestId("transformations-toolbar-icons-function-name-toggler");
+    WaitHelper.waitForElementToBeDisplayed(toggle);
+    WebElement ele = Helper.locateElementByXPath
```

Comment:
Use `Use Helper.locateElementByTestId` for locating elements by `data-testid`.

Reply:
When we are using `Helper.locateElementByTestId` we faced an Element not found Exception so to avoid that we have taken Xpath for this element